__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const request=require ('supertest')
+const test_data=require('../db/data/test-data/index')
+const seed=require('../db/seeds/seed')
+const app=require('../app')
+const db=require('../db/connection')
+
+beforeEach(()=>{
+    return seed(test_data)
+})
+
+afterAll(()=>{
+    db.end()
+})
+
+describe('GET /api/healthcheck',()=>{
+    test('returns a 200',()=>{
+        return request(app)
+            .get('/api/healthcheck')
+            .expect(200)})
+})
+
+describe('GET /api/topics',()=>{
+    // no errors expected as this is a fixed endpoint returning the whole table
+    test('returns a 200 and a list of topics with the correct data',()=>{
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body})=>{
+                expect(body.topics[0].slug).toBeDefined()
+                expect(body.topics[0].description).toBeDefined()
```

Comment:
When it comes to testing what is returned, lets be a bit more specific.
Take a look at some jest matchers, ToMatchObject or ObjectContaining would allow for better testing if you were to iterate through the response, as you are currently only looking at the first element in the response array